composer.json

```diff
@@ -128,15 +128,15 @@
         "su-sws/react_paragraphs": "dev-8.x-2.x",
         "su-sws/stanford_basic": "dev-8.x-4.x",
         "su-sws/stanford_date_formats": "dev-8.x-1.x",
-        "su-sws/stanford_events": "dev-8.x-1.x",
+        "su-sws/stanford_events": "dev-D8CORE-2431 as dev-8.x-1.x",
         "su-sws/stanford_fields": "dev-8.x-1.x",
         "su-sws/stanford_image_styles": "dev-8.x-1.x",
-        "su-sws/stanford_news": "dev-8.x-2.x",
+        "su-sws/stanford_news": "dev-D8CORE-2431 as dev-8.x-2.x",
         "su-sws/stanford_notifications": "dev-8.x-1.x",
-        "su-sws/stanford_person": "dev-8.x-1.x",
-        "su-sws/stanford_profile_helper": "dev-8.x-1.x",
+        "su-sws/stanford_person": "dev-D8CORE-2431 as dev-8.x-1.x",
+        "su-sws/stanford_profile_helper": "dev-D8CORE-2431 as dev-8.x-1.x",
         "su-sws/stanford_ssp": "dev-8.x-1.x",
-        "su-sws/stanford_text_editor": "dev-8.x-1.x"
+        "su-sws/stanford_text_editor": "dev-D8CORE-2431 as dev-8.x-1.x"
```

Comment:
Just change these back, let the tests pass and it's g2g.

Also for future `stanford_media` things, you can just add `"su-sws/stanford_media": "dev-[branch_name] as dev-8.x-1.x"` Adding that to the list of dependencies prevents you from needing all the extra branches in each repo.